src/html-transform.ts

```diff
@@ -133,7 +133,7 @@ export function htmlTransform(
         finalModuleScript.parentNode!, finalModuleScript, fragment);
   }
 
-  if (options.injectAmdLoader) {
+  if (options.injectAmdLoader && firstModuleScript !== undefined) {
```

Comment:
This is subtle and easy to see it breaking. Would a quick test be warranted?

Reply:
Done.